iptables-detect/iptables-detect.sh

```diff
@@ -171,7 +171,8 @@ os_detect() {
 
     oracleserver)
         dist_version="$(. /etc/os-release && echo "$VERSION_ID")"
-        if [ "$dist_version" -ge 8 ]; then
+        maj_ver=$(sed -E -e "s/^([0-9]+)\.?[0-9]*$/\1/" <<<$dist_version)
```

Comment:
Will this variable's regex ever diverge from the one below? Will they always be the same? Would it make sense to declare above and reuse?

Reply:
I am unsure of OEL's path forward in handling VERSION_ID, and would prefer to keep the REGEX independently defined, in the event of a future split in behavior